freetypefont.mod/freetypefont.bmx

```diff
@@ -166,7 +166,41 @@ Type TFreeTypeFont Extends BRL.Font.TFont
 				
 		Local ft_face:Byte Ptr
 
-		If src.Find( "::" )>0
+		If TStream(src)
+			Local stream:TStream = TStream(src)
+			Local data:Byte[1024]
+			Local dataSize:Int
+
+			'backup old stream position to set it back to it afterwards
+			Local oldStreamPos:int = stream.Pos()

```

Comment:
I wouldn't assume the stream is seekable. It doesn't have to be.
Is there code elsewhere that does this?

Reply:
If the stream isn't seekable it won't change anything - you cannot backup the position then - and you cannot restore it.
But in that case (non-seekable) we cannot do something to make the stream "as before" - or can we?